### PR TITLE
Feature rory

### PR DESCRIPTION
**Pull request by rorysproule1**

### CODE REVIEW

**Comment by rorysproule1** [comment]
simply brilliant

**Comment by christophermcw** [comment]
Needs to be split into more than one test

File: src/test/java/com/sampleweb/journey/HomeTests.java [hunk 1]
```diff
@@ -33,5 +31,6 @@ public void HomeDepartmentTest() throws InterruptedException {
         // click on the 4 star and up rating
         driver.findElementByXPath("//*[@id=\"p_72/419153031\"]/span/a/i/span").click();
 
+        // i may have not finished the last question but shhh
```
Comment (weesteeky) [inline src/test/java/com/sampleweb/journey/HomeTests.java]: Too many comments, not enough code.

Comment (MaaaChu) [inline src/test/java/com/sampleweb/journey/HomeTests.java]: Management will be informed about this
ლ(ಠ---益---ಠ)ლ

**Comment by EWeatherup** [comment]
here.... nice methods

**Comment by EWeatherup** [comment]
Single slash " / " : Single slash is used to create xpath with absolute  path i. e the xpath would be created to start selection from the document node/start node.

Double slash " //  " :  Double slash is used to create xpath with relative path  i. e the xpath would be created to start selection from anywhere within the document.

**Comment by BenAAndrew** [comment]
Don't even know where to start

**Comment by rorysproule1** [comment]
> Don't even know where to start

the top maybe

**Comment by BenAAndrew** [comment]
My favourite thing about your code is that it doesn't even assert anything so is pointless. Congrats

**Pull request closed without merging**